7_react/react1/week3/todo/src/border/Border.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Border = ({ child }) => {
```

Comment:
Same here, if you don't use this component, I recommend that you remove the file.